src/NUnitFramework/framework/Internal/Reflect.cs

```diff
@@ -226,5 +226,89 @@ public static object InvokeMethod( MethodInfo method, object fixture, params obj
         }
 
         #endregion
+
+#if NETSTANDARD1_3
+        /// <summary>
+        /// <para>
+        /// Selects the ultimate shadowing property just like <see langword="dynamic"/> would,
+        /// rather than throwing <see cref="AmbiguousMatchException"/>
+        /// for properties that shadow properties of a different property type.
+        /// </para>
+        /// <para>
+        /// If you request both public and nonpublic properties, every public property is preferred
+        /// over every nonpublic property. It would violate the principle of least surprise for a
+        /// derived class’s implementation detail to be chosen over the public API for a type.
+        /// </para>
+        /// </summary>
```

Comment:
Should we describe the three parameters here? (or just leave it be, since this is an internal implementation detail)

Reply:
The reason I didn't is because there is no `TypeInfo.GetProperty(string, BindingFlags)`– indeed, no BindingFlags even– on netstandard1.3. So there is no way to link the overload I want to link. If you have a suggestion of what else you'd expect to see I can put that in.

Reply:
I was thinking of something like
```
<param name="type">The type to search for the property with <paramref name="name"/>.</param>
<param name="name">The name of the property to get.</param>
<param name="bindingFlags">The specified requirements of the property.</param>
```